### ISSUE TITLE
clock module shows thousand seperator in year

### ISSUE THREAD

**Issue by Maclanky** [Alexays/Waybar#565]
I recently updated to the latest commit and now the clock module shows the year as 2,020 using the default tooltip-format.

My config for the clock module:

```
"clock": {
        // "timezone": "America/New_York",
        "tooltip-format": "{:%Y-%m-%d | %H:%M}",
        "format": "{:%Y-%m-%d | %H:%M}",
        "format-alt": "{:%Y-%m-%d}"
    },
```

**Comment by jomority** [comment]
I have the same problem. It was introduced in #560.

As a workaround, you can add `"locale": "C"` to your clock config.

However, this does not fix the displaying of fractional seconds. For example with `"format": "{:%Y-%m-%d %H:%M:%S}"` I get `2020-01-28 19:09:12.000123432`.

**Comment by honza** [comment]
> As a workaround, you can add "locale": "C" to your clock config.

Thank you!

**Comment by Xartos** [comment]
> However, this does not fix the displaying of fractional seconds. For example with `"format": "{:%Y-%m-%d %H:%M:%S}"` I get `2020-01-28 19:09:12.000123432`.

A workaround for this is to change `%S` to `%OS` in the format string

**Comment by skligys** [comment]
I think it's this bug in date library:

https://github.com/HowardHinnant/date/issues/525
https://github.com/HowardHinnant/date/commit/a1843097862aacb752a81a7e98231e9d9464a645

I patched in that change in my build, and it fixed it.

**Comment by Alexays** [comment]
Fixed with #715 unless timezone is specified.
While waiting for a [HowardHinnant/date](https://github.com/HowardHinnant/date) update.
